roles/generate_web_report/README.md

```diff
@@ -0,0 +1,16 @@
+# Generate HTML Reports from Network Facts
+
+Convert collected facts into HTML web reports using Jinja2 templates.
```

Comment:
for collection documenation please refer https://github.com/redhat-cop/network.base
for role documentation you can refer https://github.com/redhat-cop/network.base/blob/main/roles/remediate/README.md